app/src/test/java/com/yalta/viewmodel/ProfileViewModelTest.kt

```diff
@@ -26,34 +26,8 @@ class ProfileViewModelTest {
         viewModel = ProfileViewModel(FakeUserRepo(), FakeLogoutRepo(), coroutinesTestRule.testDispatcher)
     }
 
-    private fun ensureDefaults() {
-        assertEquals("root", viewModel.userName.value)
-        assertEquals("driver", viewModel.role.value)
-    }
-
-    @Test
-    fun testDefaults() {
-        ensureDefaults()
-    }
-
-    @Test
-    fun changeValueOnce() = coroutinesTestRule.testDispatcher.runBlockingTest {
-        ensureDefaults()
-        viewModel.changeValue()
-        assertEquals("root", viewModel.userName.value)
-        assertEquals("admin", viewModel.role.value)
-    }
-
-    @Test
-    fun changeValueTwice() = coroutinesTestRule.testDispatcher.runBlockingTest {
-        ensureDefaults()
-        viewModel.changeValue()
-        viewModel.changeValue()
-        ensureDefaults()
-    }
-
     @Test
-    fun logoutTest() = coroutinesTestRule.testDispatcher.runBlockingTest {
+    fun logout_test() = coroutinesTestRule.testDispatcher.runBlockingTest {
```

Comment:
And yeah, why are tests in snake case? Is there some guideline?

Reply:
renamed